web/src/todo_widget.ts

```diff
@@ -136,12 +219,13 @@ export class TaskData {
                 return undefined;
             },
 
-            inbound: (sender_id, data) => {
+            inbound: (sender_id: number | string, data: InboundData): void => {
                 // All readers may add tasks. For legacy reasons, the
                 // inbound idx is called key in the event.
-                const idx = data.key;
-                const task = data.task;
-                const desc = data.desc;
+                const safe_data = inbound_new_task_schema.parse(data);
```

Comment:
Call this `raw_data` before and `data` after, rather than `data` before and `safe_data` after.

Reply:
Sure, I have made the changes.